common-util/api/dune.js

```diff
@@ -0,0 +1,46 @@
+import {
+  PREDICTION_DAA_QUERY_ID,
+  PREDICTION_TXS_BY_AGENT_TYPE_QUERY_ID,
+} from 'common-util/constants';
+import get from 'lodash/get';
+
+const duneApiCall = async ({ queryId }) => {
+  try {
+    const response = await fetch(
+      `https://api.dune.com/api/v1/query/${queryId}/results`,
+      {
+        headers: {
+          'X-Dune-API-Key': process.env.NEXT_PUBLIC_DUNE_API_KEY,
+        },
+      },
+    );
+    const json = await response.json();
+    return json;
+  } catch (error) {
+    console.error(error);
+  }
+
+  return null;
+};
+
+export const getPredictionDaa = async () => {
+  const json = await duneApiCall({
+    queryId: PREDICTION_DAA_QUERY_ID,
+  });
+  const average = get(json, 'result.rows[0].avg_7d_active_agents_count');
```

Comment:
curious, why are you using `get` here instead of just accessing `json` directly?

Reply:
just copy-pasted from previous code for consistency...